src/containers/Interfaces/Sociogram.js

```diff
@@ -99,6 +99,7 @@ const Sociogram = (props) => {
           <NodeBucket
             id="NODE_BUCKET"
             node={nextUnplacedNode}
+            allowPositioning={allowPositioning}
```

Comment:
Just curious about what's happening here. Is this a bug that you've fixed?

Reply:
So, the NodeBucket wasn't showing up for me on Sociogram without this!